guide/install.md

```diff
@@ -108,7 +41,8 @@ state:
 Both kinds of state are intended to be accessed only by Moonfire NVR itself.
 However, the interface for adding new cameras is not yet written, so you will
```

Comment:
Oops, I forgot to update this. You can add cameras through "moonfire-nvr config" now.